xbmc/FileItem.h

```diff
@@ -743,6 +743,7 @@ class CFileItemList : public CFileItem
   CFileItemPtr Get(int iItem) const;
   const VECFILEITEMS& GetList() const { return m_items; }
   CFileItemPtr Get(const std::string& strPath) const;
+  CFileItemPtr GetFromPlaylist(int playlist) const;
```

Comment:
`std::shared_ptr<CFileItem>`

And I think, there is a special type `PLAYLIST::Id`, defined in "playlists/PlayListTypes.h" . Use this type instead of `int`, please.